SoftLayer/CLI/block/duplicate.py

```diff
@@ -54,9 +54,15 @@
               type=click.Choice(['hourly', 'monthly']),
               default='monthly',
               help="Optional parameter for Billing rate (default to monthly)")
+@click.option('--dependent-duplicate',
+              type=click.BOOL,
+              default=False,
+              help='Whether or not this duplicate will be a dependent duplicate '
+                    'of the origin volume (default to false)')
```

Comment:
`show_default=True` option will print out the default in the help message, so you don't have to specify here.

https://click.palletsprojects.com/en/7.x/options/

Reply:
as we discussed, this is not possible

Reply:
fixed in e424ce1